fenix/app/nimbus.fml.yaml

```diff
@@ -375,6 +375,50 @@ features:
         type: Int
         default: 1
 
+  translations:
+    description: The feature that allows on-device translations of web content.
+    variables:
+      main-flow-toolbar-enabled:
+        description: >
+          Show the primary toolbar entry point into the translations feature. (Translations icon on URL toolbar.)
+        type: Boolean
+        default: true
```

Comment:
Will flip the Nimbus default to false in bug [1853058](https://bugzilla.mozilla.org/show_bug.cgi?id=1853058). (Having it as a default of `true` while secret settings is still around, but the `main-flows` should probably be `false` before release.)